test/suites/integration/ami_test.go

```diff
@@ -182,6 +182,12 @@ var _ = Describe("AMI", func() {
 			env.ExpectCreatedNodeCount("==", 1)
 		})
 		It("should provision a node using the Ubuntu family", func() {
+			// Ubuntu is currently not supported on K8s version 1.27
```

Comment:
We should probably make a note that the Ubuntu SSM alias is not available on K8s 1.27 right now and that Karpenter will not work with this AMIFamily until this version is made available by Canonical